coursera/api.py

```diff
@@ -493,8 +493,14 @@ def _get_notebook_folder(self, url, jupyterId, **kwargs):
 
                 if os.path.isdir(self._course_name + "/notebook/" + head + "/") == False:
                     logging.info('Creating [{}] directories...'.format(head))
-                    os.makedirs(self._course_name + "/notebook/" + head + "/")
 
+                # head = head.replace(':', '-')
+                # To prevent problems in folder/file creation when names contain restricted symbols
+                head = clean_filename(head)
+                logging.ingo("Head: {}", head)
```

Comment:
1. inGo. Typo.
2.  Why do you need to print this to the user at all?
3. Did you look around the code? We're using "%s" in logging formatting. Be consistent with the current codebase.